src/transactions/feed/TransactionDetailsScreen.tsx

```diff
@@ -189,8 +187,7 @@ function TransactionDetailsScreen({ navigation, route }: Props) {
             >
               <View style={styles.rowContainer}>
                 <Text style={styles.blockExplorerLink}>
-                  {transactionNetwork === Network.Celo && t('viewOnCeloScan')}
-                  {transactionNetwork === Network.Ethereum && t('viewOnEthereumBlockExplorer')}
+                  {t(EXPLORER_LINK_TRANSLATION_STRINGS[transaction.networkId])}
```

Comment:
This changes the text from celo scan to celo explorer. Is this intentional?

Reply:
thanks, it was not. fortunately a test covered that as well